docs/_templates/compiler_version.tmpl

```diff
@@ -0,0 +1,24 @@
+.. -*- mode: rst -*-
+
+{% if data['compiler_version']%}
+
+Compiler version
+~~~~~~~~~~~~~~~~
+
+Specifying the compatible ISO version for a V1 module is done via the ``compiler_version`` field of the ``module.yml``
```

Comment:
If there are no changes to the compiler, the compiler_version will remain the same across ISO versions. 
```suggestion
Specifying the compatible compiler version for a V1 module is done via the ``compiler_version`` field of the ``module.yml``
```